_inc/client/plans/product-selector.jsx

```diff
@@ -215,5 +257,6 @@ export default connect( state => {
 		sitePlan: getSitePlan( state ),
 		siteRawlUrl: getSiteRawUrl( state ),
 		isFetchingData: isFetchingSiteData( state ) || isFetchingProducts( state ),
+		backupInfoUrl: getUpgradeUrl( state, 'aag-backups' ),
```

Comment:
what is `aag-backups`?  a comment here would be nice.

Reply:
I've added a comment in b2a24ec.